### ISSUE TITLE
mi silent mouse side buttons not working [bug]

### ISSUE THREAD

**Issue by kh3ldar** [archagon/sensible-side-buttons#84]
By default mouse reports it has 3 keys but it actually has 5 buttons. Side buttons simple don't work.

SensibleSideButtons 1.0.6
macOS Version: 14.4.1
Mouse Model: Mi Dual Mode Silent Edition - EAN: 6934177715457

**Comment by mtrescec** [comment]
+1

**Comment by atumas-bananamilk** [comment]
+1

**Comment by atumas-bananamilk** [comment]
Check out Steermouse, it works with Xiaomi.

https://github.com/archagon/sensible-side-buttons/issues/60

**Comment by Auxin001** [comment]
+1

**Comment by Nicholasmanu** [comment]
Yeah, it doesn't work, try USB overdrive. Works fine with more options to choose from
